configure.ac

```diff
@@ -165,7 +165,7 @@ AS_IF([test "x$enable_Bsymbolic" = "xcheck"],
 AS_IF([test "x$enable_Bsymbolic" = "xyes"], [JSON_BSYMBOLIC_LDFLAGS=-Wl[,]-Bsymbolic-functions])
 AC_SUBST(JSON_BSYMBOLIC_LDFLAGS)
 
-AX_APPEND_COMPILE_FLAGS([-Wall -Werror -Wcast-qual -Wno-error=deprecated-declarations])
+AX_APPEND_COMPILE_FLAGS([-Wall -Werror -Wcast-qual -Wno-error=deprecated-declarations -fPIC])
```

Comment:
instead move -fPIC to aconfigure.sh - using CFLAGS input to the ./configure statement?

That way you don't change anything in the original build files.

Reply:
Wouldn't that still cause issues while building a Linux static library for COCO SDK or any other library using this as a dependency?

Reply:
We can add the same on the external lconfigure.sh build script.

In general its best not to change 3rd party code as far as possible. It allows us to upgrade easily without remembering to "port" our changes across various versions.